src/md80.cpp

```diff
@@ -120,6 +120,11 @@ void Md80::setMaxVelocity(float _maxVelocity)
 	maxVelocity = _maxVelocity;
 	maxVelocityAdjusted = true;
 }
+void Md80::setMaxAccelerationAndDeceleration(float _maxAcceleration)
+{
+	maxAcceleration = _maxAcceleration;
+	maxAccelerationAndDecelerationAdjusted = true;
```

Comment:
seems the deceleration is not being adjusted :P the name can be changed to maxAccelerationAdjusted (same goes to the function name)